packages/office-ui-fabric-react/src/components/Layer/Layer.styles.ts

```diff
@@ -0,0 +1,38 @@
+import { ILayerStyleProps, ILayerStyles } from './Layer.types';
+import {
+  IStyle,
+  ITheme,
+} from '../../Styling';
+
+export const getStyles = (
+  props: ILayerStyleProps
+): ILayerStyles => {
+  const {
+    className,
+    theme,
+    isNotHost
+  } = props;
+
+  // const { palette, semanticColors } = theme;
+
+  return ({
+    root: [
+      'ms-Layer',
+      isNotHost && 'ms-Layer--fixed' && {
+        position: 'fixed',
+        zIndex: 1000000,
+        top: 0,
+        left: 0,
+        width: '100vw',
+        height: '100vh',
+        visibility: 'hidden'
+      }
+    ],
```

Comment:
missing className at the end here.

Reply:
If we have a snapshot test, it would be great to update it and pass a `className` through so it shows up.